src/components/how/styledHow.js

```diff
@@ -24,11 +31,27 @@ export const HowTitle = styled.h2`
   line-height: 52px;
   letter-spacing: 0em;
   margin-bottom: 42px;
+
+  @media( max-width: 1023px) {
+    text-align: center;
+  }
+
+  @media( max-width: 767px) {
```

Comment:
This is just a preference thing, but I think the font-size drops too much at 767.  I would put an intermediate size in there until 425.

Reply:
I made it so at 767 it jumps down to 36px then at 425 down to 28px. I think it looks better now. Good call